src/components/Chat.js

```diff
@@ -0,0 +1,214 @@
+import 'moment/locale/de';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Avatar } from 'react-native-elements';
+import {
+  Bubble,
+  Composer,
+  GiftedChat,
+  InputToolbar,
+  MessageText,
+  Send
+} from 'react-native-gifted-chat';
+
+import { colors, Icon, normalize } from '../config';
+import { momentFormat } from '../helpers';
+
+import { RegularText } from './Text';
+
+/* eslint-disable react-native/no-inline-styles */
+/* eslint-disable react/prop-types */
```

Comment:
i will remove these and fix the nline style with adding a style and the prop types with adding the missing prop type for `currentMessage`